### ISSUE TITLE
About the results of CoNLL 2003 NER

### ISSUE THREAD

**Issue by wangxinyu0922** [studio-ousia/luke#19]
Hi, I'm very interested in your great work! In the paper, results are reported based on document-level context following BERT. Since a lot of previous work (including some of the results in the paper) used sentence-level context only, I'm wondering how LUKE performs on the CoNLL NER dataset with sentence-level context. It will be very helpful if you can report the results based on the sentence-level context.

**Comment by ikuyamada** [comment]
Hi, Thank you for your interest to our work! I am planning to work on this repository soon, and run an experiment based on the sentence-level context :)

**Comment by ikuyamada** [comment]
As there is no activity, I close this issue.

**Comment by wangxinyu0922** [comment]
> Hi, Thank you for your interest to our work! I am planning to work on this repository soon, and run an experiment based on the sentence-level context :)

Hi, do you have any results for luke based on sentence-level context?

**Comment by wangxinyu0922** [comment]
I tried to run the dataset with sentence-level through adding `-DOCSTART-` between every two sentence to train a sentence-level model and finally got `92.07` F1 score without any parameter tuning. Do you have any suggestion to improve the score?

**Comment by ikuyamada** [comment]
I am sorry for the delayed response, and thank you for reporting the sentence-level NER results!
I think it might be useful to observe how the results would be changed when activating `--no-word-feature` or `--no-entity-feature` options (addressing the task only using word or entity embeddings).

**Comment by wangxinyu0922** [comment]
Hi, I tried to use the two options and the results are as follows:
`--no-word-feature`: 90.9
`--no-entity-feature`: 91.2

It seeems that both word and entity features help the model accuracy.

**Comment by ikuyamada** [comment]
Hi, Thank you for conducting extra experiments and reporting the results!
These results are consistent with our experimental results.
Regarding the performance, since we conducted pretraining only using long word sequences (`max_seq_length=512`), our model may not be good at capturing the information of entities based on short sentence-level contexts.

**Comment by wangxinyu0922** [comment]
Thank you for the explanation! It helps me to better understand the strength of LUKE for future experiments.
